### PR TITLE
EID-601 Create a set of test tools scenarios to cover eIDAS identities

### PR DESCRIPTION
**Pull request by chrisgrimble**
Co-authored-by: Minno Dang <minh.ngocdang@digital.cabinet-office.gov.uk>

### CODE REVIEW

File: verify-matching-service-test-tool/src/main/resources/eidas/special-name-case.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          "verified": true
+        }
+      ],
+      "dateOfBirth": {
+        "value": "1980-05-24",
+        "verified": true
+      },
+      "firstName": {
+        "value": "Joá",
+        "nonLatinScriptValue": "Joá",
```
Comment (simonwo) [inline verify-matching-service-test-tool/src/main/resources/eidas/special-name-case.json]: Does that á count as a Latin character? Do we have some reference which lists what standard we are applying for Latin/non-Latin characters?

File: verify-matching-service-test-tool/src/main/resources/eidas/special-name-case.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "verified": true
+      },
+      "gender": {
+        "value": "MALE",
+        "verified": true
+      },
+      "middleNames": {
+        "value": "Bob Rob",
+        "nonLatinScriptValue": "Γεώργιος",
```
Comment (simonwo) [inline verify-matching-service-test-tool/src/main/resources/eidas/special-name-case.json]: That Greek name is a Greek equivalent of "George". Can we update the `value` to be "George" or "Georgios"?

File: verify-matching-service-test-tool/src/main/resources/legacy/special-name-case.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+  {
+    "hashedPid": "8fbfdd035a92daf890673d9ff8f50def3aad66ea42026c581bd73735250ba706",
+    "levelOfAssurance": "LEVEL_1",
+    "matchId": "default-match-id",
+    "matchingDataset": {
+      "addresses": [
+        {
+          "internationalPostCode": "GB1 2PF",
```
Comment (simonwo) [inline verify-matching-service-test-tool/src/main/resources/legacy/special-name-case.json]: Is this a deliberate choice based on a decision about how we'll use `internationalPostCode`? Last time I spoke about it was said we would leave it blank...

File: verify-matching-service-test-tool/src/main/resources/legacy/special-name-case.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+          "lines": [
+            "10 George Street"
```
Comment (simonwo) [inline verify-matching-service-test-tool/src/main/resources/legacy/special-name-case.json]: Is this right? I'd expect to see more `lines` here.

File: verify-matching-service-test-tool/src/main/resources/legacy/special-name-case.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        "nonLatinScriptValue": "Joá",
+        "verified": true
+      },
+      "gender": {
+        "value": "MALE",
+        "verified": true
+      },
+      "middleNames": {
+        "value": "Bob Rob",
+        "nonLatinScriptValue": "Γεώργιος",
```
Comment (simonwo) [inline verify-matching-service-test-tool/src/main/resources/legacy/special-name-case.json]: Do we actually support non-Latin names in the Verify-flavour case?

**Comment by simonwo** [comment]
- [ ] Remove `nonLatinScriptValue` for legacy case
- [ ] Make sure it's `fromDate` and `toDate` in legacy case

**Comment by chrisgrimble** [comment]
This has now been superseded by EID-615

**Pull request closed without merging**